tests/test_chi_ssa_61.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest  # noqa
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ssa_61 import ChiSsa61Spider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "chi_ssa_61.html"),
+    url="http://www.downtownhydeparkchicago.com/about/",
+)
+spider = ChiSsa61Spider()
+
+freezer = freeze_time("2019-10-04")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
```

Comment:
We'll want to include all of the existing tests from the template here, not because there's much notable happening but as regression tests to make sure we're aware of anything that will change if we have to update the scraper